pipeline_tools/pipelines/optimus/optimus.py

```diff
@@ -185,3 +188,25 @@ def create_optimus_input_tsv(uuid, version, dss_url):
         f.write(f"{chemistry}")
 
     print('Finished writing files')
+
+
+def create_optimus_inputs_tsv_from_analysis_metadata(metadata_file):
```

Comment:
Does this metadata file contain the Cromwell metadata? A docstring here would help!

Reply:
Another good question - updating the docstring to make it clear that this is Cromwell metadata.